app/code/Magento/Inventory/Model/SourceSelectionInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+interface SourceSelectionInterface
+{
+    /**
+     * @param $store
+     * @param $items
+     * @param \Magento\Quote\Model\Quote\Address $destinationAddress
+     * @return Package[]
+     */
+    public function getPackages($store, $items, $destinationAddress);
```

Comment:
is this method supposed to be called several times in case of multi-shipping delivery?